src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Security\InteractiveSecurityHelper;
+use Symfony\Bundle\MakerBundle\Util\YamlSourceManipulator;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use SymfonyCasts\Bundle\ResetPassword\SymfonyCastsResetPasswordBundle;
+
+class MakeResetPassword extends AbstractMaker
+{
+    /**
+     * @var FileManager
+     */
+    private $fileManager;
+
+    public function __construct(FileManager $fileManager)
+    {
+        $this->fileManager = $fileManager;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:reset-password';
+    }
+
+    public function configureCommand(
+        Command $command,
+        InputConfiguration $inputConfig
+    ) {
+        $command
+            ->setDescription('Create controller, entity, and repositories for use with SymfonyCasts Reset Password Bundle.')
+        ;
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+        $dependencies->addClassDependency(SymfonyCastsResetPasswordBundle::class, 'symfonycasts/reset-password-bundle');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $io->title('Reset Password Bundle Requires:');
+        $requirements[] = '1) A user entity has been created.';
+        $requirements[] = '2) The user entity contains an email property with a getter method.';
+        $requirements[] = '3) A user repository exists for the user entity.'."\n";
+        $requirements[] = '<fg=yellow>bin/console make:user</> will generate the user entity and it\'s repository...'."\n";
+        $io->text($requirements);
+
+        // initialize arguments & commands that are internal (i.e. meant only to be asked)
+        $command
+            ->addArgument('from-email-address', InputArgument::REQUIRED)
+            ->addArgument('from-email-name', InputArgument::REQUIRED)
+            ->addArgument('controller-reset-success-redirect', InputArgument::REQUIRED)
+            ->addArgument('user-class')
+            ->addArgument('email-property-name')
+            ->addArgument('email-getter')
+            ->addArgument('password-setter')
+        ;
+
+        $io->section('- Email Templates -');
```

Comment:
My initial instinct is that this is cool :)

Reply:
hmm, off the top of my head, all of these args are "required". Do we need `InputArgument::REQUIRED` for the last 4 even though the interactive security guesser sort of "enforces" this..